packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1072,6 +1072,12 @@ contract OPContractsManagerDeployer is OPContractsManagerBase {
             output.opChainProxyAdmin, address(output.systemConfigProxy), implementation.systemConfigImpl, data
         );
 
+        // If the custom gas token feature was requested, enable the custom gas token feature in the SystemConfig
+        // contract.
+        if (_input.useCustomGasToken) {
```

Comment:
Although this works, it'd be nicer if we could configure system customizations like this in a more generic way (e.g., an array of string names, or a bitmap, or something similar). If we go with this pattern then we're going to have breaking changes to the input ABI every time we add a new thing. (useCustomGasToken, useEthLockbox, useCgt). Each change to the OPCM ABI creates a bunch of problems. We do a bitmap for dev features (`devFeatureBitmap`), op-deployer probably supports arrays well enough now that we can use an array here instead, so you'd supply something like: `_input.enabledSystemFeatures: ["CUSTOM_GAS_TOKEN", "ETH_LOCKBOX"]`.